docs/USAGE-GUIDE.md

```diff
@@ -251,6 +251,29 @@ typedef enum { S2N_STATUS_REQUEST_NONE, S2N_STATUS_REQUEST_OCSP } s2n_status_req
 status request an S2N_CLIENT should make during the handshake. The only
 supported status request type is OCSP, **S2N_STATUS_REQUEST_OCSP**.
 
+
+```c
+typedef enum { S2N_CERT_AUTH_NONE, S2N_CERT_AUTH_REQUIRED } s2n_cert_auth_type;
+```
+**s2n_cert_auth_type** is used to declare what type of client certificiate authentication to use.
+Currently the default for s2n is for neither the server side or the client side to use Client (aka Mutual) authentication.
+
```

Comment:
I think renaming `s2n_cert_auth_type` to `s2n_client_auth_type` may be more clear. 

I don't think we want to change this to be peer authentication since we don't want to allow users to configure s2n Clients to skip the authentication of the Server's Cert.

Agree with the default settings. Default should be to only Authenticate the Server's Cert with no Client Authentication.